drizzlepac/haputils/catalog_utils.py

```diff
@@ -1746,15 +1746,15 @@ def detect_and_eval_segments(self, imgarr, kernel, ncount, size_source_box, nsig
             if segm_img is None:
                 log.warning("End processing for the segmentation catalog due to no sources detected with the current kernel.")
                 log.warning("No segmentation catalog will be produced for this total detection product, {}.".format(self.imgname))
-                is_big_crowded = False
-                big_island = 100.0
-                source_fraction = 100.0
+                is_big_crowded = True
+                big_island = 1.0
+                source_fraction = 1.0
```

Comment:
The logic here for 'seg_img is None' duplicates the logic at the start of the method '_evaluate_segmentation_image'.  Removing this entire if...else block and adding these log messages to the method's own 'if seg_img is None' block of code would simplify the code by putting this check in only one place.

Reply:
Does not hurt anything.